### ISSUE TITLE
Error serving build output: serve is not a function

### ISSUE THREAD

**Issue by decanTyme** [percy/gatsby-plugin-percy#83]
## The problem

I was following along with the docs in how to integrate to Gatsby [here](https://docs.percy.io/docs/gatsby). Running the command `yarn percy exec -- gatsby build` on the usage section, I get the following error:

~~~
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
[percy] Warning: Missing command separator (--), some command options may not work as expected
[percy] Percy has started!
[percy] Running "gatsby build"
success compile gatsby files - 0.947s
...

ERROR 

Error serving build output:
 serve is not a function



  TypeError: serve is not a function

  - gatsby-node.js:73 
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:73:12        

  - gatsby-node.js:26 Object.context.try
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:26:20        

  - gatsby-node.js:72 Object.exports.onPostBuild
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:72:34        

  - runMicrotasks

  - task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5

  - api-runner-node.js:462 runAPI
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby]/src/utils/api-runner-node.js:462:16      


success Percy - 0.110s - Serving build output...
~~~

## Environment

- Node version: **17.6.0**
- `@percy/cli` version: **1.0.0-beta.76**
- Version of Percy SDK you’re using: 
- If needed, a build or snapshot ID:
- OS version: **Windows 10 10.0.19043**
- Type of shell command-line [interface]: **Git Bash**

## Details

I have tried everything including linking the repo to the project. Examining the actual code in the plugin, I found out that `@percy/cli-snapshot/dist/utils` doesn't actually contain the code for `serve()` (**!**). If I'm not mistaken this is what's entirely causing the error.

```js
// https://github.com/percy/gatsby-plugin-percy/blob/master/gatsby-node.js 
53 | let { serve, mapStaticSnapshots } = require('@percy/cli-snapshot/dist/utils');
```

I just wanna try using Percy but the error is keeping me from uploading snapshots so the build always fails on the dashboard.

## Debug logs

~~~
$ npx percy exec --debug -- gatsby build
[percy:config] Config file not found (0ms)
[percy:core:browser] Launching browser (17ms)
[percy:core:browser] Browser connected [41168]: HeadlessChrome/92.0.4515.0 (171ms)
[percy:core] Percy has started! (2ms)
[percy:cli:exec] Running "gatsby build" (1ms)
success compile gatsby files - 1.415s
success load gatsby config - 0.035s
success load plugins - 1.298s
success onPreInit - 0.004s
success delete worker cache from previous builds - 0.003s
info One or more of your plugins have changed since the last time you ran Gatsby. As
a precaution, we're deleting your site's cache to ensure there's no stale data.
success initialize cache - 0.069s
success copy gatsby files - 0.160s
success Compiling Gatsby Functions - 0.201s
success onPreBootstrap - 0.211s
success createSchemaCustomization - 0.008s
success Checking for changed pages - 0.001s
success source and transform nodes - 0.159s
info Writing GraphQL type definitions to D:/Projects/Web
Development/sample-netlify-cms-icp-ep.se-ustp-site/.cache/schema.gql
success building schema - 0.276s
success createPages - 0.001s
success createPagesStatefully - 0.127s
info Total nodes: 99, SitePage nodes: 10 (use --verbose for breakdown)
success Checking for changed pages - 0.001s
success Cleaning up stale page-data - 0.002s
success onPreExtractQueries - 0.001s
success extract queries from components - 3.449s
success write out redirect data - 0.005s
success Build manifest and related icons - 0.134s
success onPostBootstrap - 0.139s
info bootstrap finished - 9.905s
success write out requires - 0.006s
success Building production JavaScript and CSS bundles - 23.656s
success Building HTML renderer - 8.767s
success Execute page configs - 0.018s
success Caching Webpack compilations - 0.001s
success run queries in workers - 0.228s - 10/10 43.80/s
success Merge worker state - 0.018s
success Rewriting compilation hashes - 0.002s
success Writing page-data.json files to public directory - 0.017s - 10/10 599.59/s
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
warn [gatsby-plugin-image] Missing image prop
success Building static HTML for pages - 2.917s - 10/10 3.43/s
info [gatsby-plugin-netlify] Creating SSR redirects...
info [gatsby-plugin-netlify] Created 0 SSR redirects...
info [gatsby-plugin-netlify] No Netlify functions needed. Skipping...

 ERROR 

Error serving build output:
 serve is not a function



  TypeError: serve is not a function

  - gatsby-node.js:73 
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:73:12        

  - gatsby-node.js:26 Object.context.try
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:26:20        

  - gatsby-node.js:72 Object.exports.onPostBuild
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby-plugin-percy]/gatsby-node.js:72:34        

  - runMicrotasks

  - task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5

  - api-runner-node.js:462 runAPI
    [sample-netlify-cms-icp-ep.se-ustp-site]/[gatsby]/src/utils/api-runner-node.js:462:16      


success Percy - 0.111s - Serving build output...
success onPostBuild - 0.142s
info Done building in 46.7980673 sec


Pages

┌ src/pages/404.js
│ ├   /404
│ └   /404.html
├ src/pages/index.js
│ └   /
├ src/pages/blog/index.js
│ └   /blog
└ src/pages/blog/{MarkdownRemark.frontmatter__title}.js
  ├   /blog/test-4
  └   ...5 more pages available

  ╭────────────────────────────────────────────────────────────────╮
  │                                                                │
  │   (SSG) Generated at build time                                │
  │ D (DSG) Deferred static generation - page generated at runtime │
  │ ∞ (SSR) Server-side renders at runtime (uses getServerData)    │
  │ λ (Function) Gatsby function                                   │
  │                                                                │
  ╰────────────────────────────────────────────────────────────────╯

[percy:core:browser] Closing browser (47145ms)
[percy:core:browser] Browser closed (115ms)
[percy:core] Build not created (0ms)
~~~

## Code to reproduce issue

[This is a repo](https://github.com/decanTyme/sample-netlify-cms-icp-ep.se-ustp-site) intended to be used as an example for our upcoming school org website and the one I was experimenting Percy with. Any branches will reproduce the issue when running `yarn percy exec -- gatsby build`. You'll also see in the [latest PR](https://github.com/decanTyme/sample-netlify-cms-icp-ep.se-ustp-site/pull/18) why Percy fails.

**Comment by rrodwell** [comment]
Any update on this?

**Comment by Robdel12** [comment]
👋🏼 See https://github.com/percy/gatsby-plugin-percy/issues/102#issuecomment-1154261124 since it's the same issue

**Comment by Robdel12** [comment]
Fixed with v1.1.0 🥳
